hecuba_py/tests/withcassandra/tutorial_tests.py

```diff
@@ -6,67 +6,72 @@
 
 class ExampleStoragedictClass(StorageDict):
     '''
-        @TypeSpec <<int>, str>
+        @TypeSpec table1 dict<<k1:int>, val1:str>
```

Comment:
@Riuzan 

Is it mandatory to give a name to a `@TypeSpec` ? Can you adapt the parser to not require a name for the StorageDict?

Reply:
The point is having a name for the StorageObj attributes (`@ClassField`) makes sense. Setting a name before the dictionary does not.

Reply:
It can be adapted, Ismael is working on it now.